src/app/components/media/MediaBanner.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import Person from '../../icons/person.svg';
+import PersonAdd from '../../icons/person_add.svg';
+import PersonCheck from '../../icons/person_check.svg';
+import Bolt from '../../icons/bolt.svg';
+import Tipline from '../../icons/question_answer.svg';
+import Alert from '../cds/alerts-and-prompts/Alert';
+import CheckMediaOrigin from '../../CheckMediaOrigin';
+
+const getIconAndMessage = (type, user, cluster, datetime) => {
+  // The default messages are temporary, and will be updated in the ticket CV2-5785
+  switch (type) {
+  case CheckMediaOrigin.TIPLINE_SUBMITTED:
+    return {
+      icon: <Tipline />,
+      message: (
+        <FormattedHTMLMessage
+          defaultMessage="This media was submitted via <strong>Tipline</strong> on {datetime}."
+          description="Message for Tipline Submitted"
+          id="MediaBanner.tiplineSubmitted"
```

Comment:
Convention for string ids should be `mediaBanner.stringIdentifier`

Reply:
:white_check_mark: